packages/trpc/src/test/recipe.spec.js

```diff
@@ -0,0 +1,115 @@
+import { trpcSetup, tearDown } from "./utils";
+import { prisma } from "@recipesage/prisma";
+
+describe("Recipes", () => {
+  let user;
+  let trpc;
+
+  beforeAll(async () => {
+    ({ user, trpc } = await trpcSetup());
+  });
+
+  afterAll(() => {
+    return tearDown(user.id);
+  });
+
+  it("creates a recipe", async () => {
+    const recipe = await trpc.recipes.createRecipe.mutate({
+      title: "Sliced cucumber",
```

Comment:
For all of these values, I'd love for us to use the library [faker.js](https://fakerjs.dev/) to randomize the values